src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`styles/main.scss`). HTML does not support SCSS directly. You need to compile the SCSS file into a CSS file and then link to the resulting CSS file instead.